src/Pim/Bundle/CatalogBundle/spec/Elasticsearch/Filter/OptionFilterSpec.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+namespace spec\Pim\Bundle\CatalogBundle\Elasticsearch\Filter;
+
+use Akeneo\Component\StorageUtils\Exception\InvalidPropertyException;
+use Akeneo\Component\StorageUtils\Exception\InvalidPropertyTypeException;
+use Documents\Ecommerce\Option;
+use PhpSpec\ObjectBehavior;
+use Pim\Bundle\CatalogBundle\Elasticsearch\Filter\AbstractAttributeFilter;
+use Pim\Bundle\CatalogBundle\Elasticsearch\Filter\OptionFilter;
+use Pim\Bundle\CatalogBundle\Elasticsearch\SearchQueryBuilder;
+use Pim\Component\Catalog\Exception\InvalidOperatorException;
+use Pim\Component\Catalog\Model\AttributeInterface;
+use Pim\Component\Catalog\Query\Filter\AttributeFilterInterface;
+use Pim\Component\Catalog\Query\Filter\Operators;
+use Pim\Component\Catalog\Validator\AttributeValidatorHelper;
+
+class OptionFilterSpec extends ObjectBehavior
+{
+    function let(AttributeValidatorHelper $attributeValidatorHelper)
+    {
+        $operators = ['IN', 'EMPTY', 'NOT_EMPTY', 'NOT IN'];
+        $this->beConstructedWith($attributeValidatorHelper, ['pim_catalog_simpleselect'], $operators);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(OptionFilter::class);
+    }
+
+    function it_is_a_filter()
+    {
+        $this->shouldImplement(AttributeFilterInterface::class);
+        $this->shouldBeAnInstanceOf(AbstractAttributeFilter::class);
+    }
+
+    function it_supports_operators()
+    {
+        $this->getOperators()->shouldReturn(
```

Comment:
Here you describe that the class exposes the configured operators, this should go in another example.